### ISSUE TITLE
Fritz SP Extension: Add the ability to fit sncosmo models to photometry

### ISSUE THREAD

**Issue by dannygoldstein** [fritz-marshal/fritz#92]
Request from Kate M: 

Add the ability to fit SALT2 SN Ia model to a user's visible photometry.

**Comment by cfremling** [comment]
i see this is not assigned to anyone. The checking against SN Ia LCs is an integral part of the RCF classification workflow.

**Comment by dmitryduev** [comment]
@cfremling do you want to try implementing this and committing to the codebase? I assume you have the code to do this manually -- we'd be happy to help out integrating that code with Fritz.

**Comment by kmshin1397** [comment]
@guynir42 Can you remind us what your thoughts were on fitting/modeling? I know you were working on it for a bit a few months ago

**Comment by guynir42** [comment]
@kmshin1397 I think what we talked about was that (a) fitting is too specific to run on SkyPortal, and should be deferred to an external service and (b) once the fit is done the fit result should be posted as an annotation. 

The annotation should be formatted in some agreed-upon way that makes it show up as a "graphical annotation". 
I don't know what that means exactly, but it must include x and y values (and errors?) provided by the user, and plotted on top of the spectrum (or photometry for that matter).

**Comment by mcoughlin** [comment]
Being dealt with in SP.
